### DISCUSSION TITLE
Installed, but got HTTP 400 BAD REQUEST

### DISCUSSION THREAD

**Discussion by parsh78** [trypromptly/LLMStack#58] [Q&A] [answered]
Am very excited to try out this tool. However after installing, I noticed that all the containers are running but I am not able to access using http://192.168.1.39:3000. It responds with HTTP 400 Bad request error.

So sshed into this VM and did a CURL using localhost:3000 and it seems to be working. Please can you guide how to make it work with IP address as well? I am installing on a VM on network.

thank you!

**Comment by ajhai** [comment, marked as answer]
@parsh78 can you add `ALLOWED_HOSTS` and `CSRF_TRUSTED_ORIGINS` to your environment? See https://llmstack.ai/docs/getting-started/introduction#:~:text=If%20you%20are%20deploying%20LLMStack%20on%20a%20server%2C%20make%20sure%20to%20update%20ALLOWED_HOSTS%20and%20CSRF_TRUSTED_ORIGINS%20in%20.env%20file%20to%20include%20the%20hostname%20of%20your%20server.%20Refer%20to%20the%20configuration%20section%20for%20more%20information.
